docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1883 @@
+{
```

Comment:
<p>Emphasize that you are talking about the first input (rng) and also mention now you changed size as well. </p><p> </p><p> </p><p>Also a bit strange to first show pm.Normal.dist and then at.random.normal. Maybe the other way around? Or show both examples with at.random.normal and quickly mention you get the same object when you call pm.Normal.dist?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
I tried to rephrase it following your suggestion. Feedback is welcome :)